bin/oref0-pump-loop.sh

```diff
@@ -348,11 +348,17 @@ function mmtune {
 }
 
 function maybe_mmtune {
-    # mmtune ~ 25% of the time
-    [[ $(( ( RANDOM % 100 ) )) > 75 ]] \
-    && echo "Waiting for 30s silence before mmtuning" \
-    && wait_for_silence 30 \
-    && mmtune
+    if ( find monitor/ -mmin -15 -size +5c | egrep -q "mmtune|pump_loop_completed" ); then
+        # mmtune ~ 25% of the time
+        [[ $(( ( RANDOM % 100 ) )) > 75 ]] \
+        && echo "Waiting for 30s silence before mmtuning" \
+        && wait_for_silence 30 \
+        && mmtune
+    else
+        echo "mmtune.json and pump_loop_completed more than 15m old; waiting for 30s silence before mmtuning"
```

Comment:
according to the the find command this should be `mmtune.json or pump_loop_completed more than 15m old`

Reply:
if find mmtune or loop_completed <15m old, randomize. 
else, both mmtune and loop_completed are >15m old, so mmtune.